eng/WpfArcadeSdk/tools/Packaging.targets

```diff
@@ -120,6 +120,16 @@ $(PreparePackageAssetsDependsOn):
                 Condition="'$(DestinationSubFolder)'!='' and '$(IsPackagingProject)'!='true' and '@(PackagingAssemblyContent)'==''">
       <Output ItemName="PackageAsset" TaskParameter="Include"/>
     </CreateItem>
+    <CreateItem Include="$(OutDir)*.deps.json"
+                AdditionalMetadata="RelativePath=$(ArtifactsPackagingDir)$(NormalizedPackageName)\$(DestinationSubFolder)"
+                Condition="'$(DestinationSubFolder)'!='' and '$(IsPackagingProject)'!='true' and '@(PackagingAssemblyContent)'==''">
```

Comment:
I’d rather these weren’t added globally, and instead these were added by projects that need them as PackagingContent. 

I won’t have an objection to the introduction of an extensibility point for `PreparePackageAssets`, where about test specific target globally adds more items to `PackageAsset`.

Reply:
huh, this is odd, i don't remember adding this. i wonder if there was a bad merge or something on my end

Reply:
:)

Reply:
ohhh nvm, why are these considered different then everything else?

Reply:
Separation of concerns. These targets exist to serve the needs to shipping product. Needs of tests can either be served by them as-is, or using an extensibility.